### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by AbigiyaTY**
# Changes we made in this PR:

- Add landing pages with the list of items from the API
- Add a comment section for each items
- Add reservation section for each item

### CODE REVIEW

File: src/mealList.js [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+    resevationContainer.appendChild(rsvButton);
+    listItem.appendChild(resevationContainer);
+
+    foodDiv.appendChild(listItem);
+    mainDiv.appendChild(foodDiv);
+
+    foodDiv.appendChild(listItem);
+    mainDiv.appendChild(foodDiv);
+
+    // eslint-disable-next-line no-await-in-loop
```
Comment (Deepakdanger) [inline src/mealList.js:95]: - [ ] Kindly remove this line, & solve the linter manually.

File: src/mealList.js [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+    let date_end = document.querySelector('#Enddate').value;
+    const itemId = meal.idMeal;
+    const newData = {
+      item_id: itemId,
+      username,
+      date_start,
+      date_end,
+    };
+    postResevation(newData);
+    /* eslin-disablet no-undef */
```
Comment (Deepakdanger) [inline src/mealList.js:292]: - [ ] Kindly remove this line & solve the linter manually.

File: src/mealList.js [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+      <button type="button" class="btn btn-secondary reservationBtn  my-2">Reserve</button>
+ </li>
+</ul>
+    </form>
+  `;
+  mealReseverDetails.parentElement.classList.add('showResevation');
+  const commentBtn = document.querySelector('.reservationBtn');
+  commentBtn.addEventListener('click', () => {
+    let username = document.querySelector('#commentator').value;
+    /* eslint-disable camelcase */
```
Comment (Deepakdanger) [inline src/mealList.js:281]: - [ ] Kindly remove this line & solve the linter manually.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
**Hi @TEAM**, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
 - [ ] In reservation creation, the `Name` is not displayed in the reservation list.KIndy fixes it.
Screenshot from 2022-09-10 16-46-08

- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/mealList.js (hunk 1 above)
Comment (tufoinnkuo10) [inline src/mealList.js:95]: - [ ] please like the previous reviewer noted, eslint is necessary with the goal of making code more consistent and avoiding bugs. so disabling it only means there is an issue. remove and solve the error like he said. great job

File: src/mealList.js [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
```
Comment (tufoinnkuo10) [inline src/mealList.js:281]: - [ ] here as well

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
## Hi @Team

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ site looks professional

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @Loltolo-Lesapiti, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well implemented suggested changes

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

> 

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**